src/install.ml

```diff
@@ -18,109 +20,128 @@ module Section = struct
+    | Libexec_root -> "libexec_root"
+    | Bin          -> "bin"
+    | Sbin         -> "sbin"
+    | Toplevel     -> "toplevel"
+    | Share        -> "share"
+    | Share_root   -> "share_root"
+    | Etc          -> "etc"
+    | Doc          -> "doc"
+    | Stublibs     -> "stublibs"
+    | Man          -> "man"
+    | Misc         -> "misc"
 
   let of_string = function
-    | "lib"        -> Some Lib
-    | "libexec"    -> Some Libexec
-    | "bin"        -> Some Bin
-    | "sbin"       -> Some Sbin
-    | "toplevel"   -> Some Toplevel
-    | "share"      -> Some Share
-    | "share_root" -> Some Share_root
-    | "etc"        -> Some Etc
-    | "doc"        -> Some Doc
-    | "stublibs"   -> Some Stublibs
-    | "man"        -> Some Man
-    | "misc"       -> Some Misc
-    | _            -> None
+    |"lib"          -> Some Lib
+    |"lib_root"     -> Some Lib_root
+    |"libexec"      -> Some Libexec
+    |"libexec_root" -> Some Libexec_root
+    |"bin"          -> Some Bin
+    |"sbin"         -> Some Sbin
+    |"toplevel"     -> Some Toplevel
+    |"share"        -> Some Share
+    |"share_root"   -> Some Share_root
+    |"etc"          -> Some Etc
+    |"doc"          -> Some Doc
+    |"stublibs"     -> Some Stublibs
+    |"man"          -> Some Man
+    |"misc"         -> Some Misc
+    | _             -> None
 
   let t =
     let open Sexp.Of_sexp in
     enum
-      [ "lib"        , Lib
-      ; "libexec"    , Libexec
-      ; "bin"        , Bin
-      ; "sbin"       , Sbin
-      ; "toplevel"   , Toplevel
-      ; "share"      , Share
-      ; "share_root" , Share_root
-      ; "etc"        , Etc
-      ; "doc"        , Doc
-      ; "stublibs"   , Stublibs
-      ; "man"        , Man
-      ; "misc"       , Misc
+      [ "lib"          , Lib
+      ; "lib_root"     , Lib_root
+      ; "libexec"      , Libexec
+      ; "libexec_root" , Libexec_root
+      ; "bin"          , Bin
+      ; "sbin"         , Sbin
+      ; "toplevel"     , Toplevel
+      ; "share"        , Share
+      ; "share_root"   , Share_root
+      ; "etc"          , Etc
+      ; "doc"          , Doc
+      ; "stublibs"     , Stublibs
+      ; "man"          , Man
+      ; "misc"         , Misc
       ]
 
   let should_set_executable_bit = function
-    | Lib        -> false
-    | Libexec    -> true
-    | Bin        -> true
-    | Sbin       -> true
-    | Toplevel   -> false
-    | Share      -> false
-    | Share_root -> false
-    | Etc        -> false
-    | Doc        -> false
-    | Stublibs   -> true
-    | Man        -> false
-    | Misc       -> false
+    | Lib          -> false
+    | Lib_root     -> false
+    | Libexec      -> true
+    | Libexec_root -> true
+    | Bin          -> true
+    | Sbin         -> true
+    | Toplevel     -> false
+    | Share        -> false
+    | Share_root   -> false
+    | Etc          -> false
+    | Doc          -> false
+    | Stublibs     -> true
+    | Man          -> false
+    | Misc         -> false
```

Comment:
Would be cleaner if the false/true rhs were shared.

Reply:
Indeed, I made the change